### PR TITLE
GFORMS-2958 - Disassociate miniSummaryList labels with En/Cy mandator…

### PR DESCRIPTION
**Pull request by DursunGundogan**
…y label/titles

### CODE REVIEW

File: app/uk/gov/hmrc/gform/formtemplate/FormTemplateValidator.scala [hunk 1]
```diff
@@ -784,9 +784,14 @@ object FormTemplateValidator {
...
 
+    def ignoredComponent(fc: FormComponent) = fc match {
+      case IsInformationMessage(_) | IsMiniSummaryList(_) | IsTableComp(_) => true
+      case _                                                               => false
+    }
+
     val validationResults = sectionsList.flatMap { page =>
       (page.allFormComponents ++ page.fields).collect {
-        case fc if isEmptyLabel(fc.label) && isEmptyCy(page.title) =>
+        case fc if !ignoredComponent(fc) && isEmptyLabel(fc.label) && isEmptyCy(page.title) =>
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/formtemplate/FormTemplateValidator.scala:794]: Why this branch even exists? Why are we checking empty welsh label?

**Review by josef-vlach — COMMENTED** [review]
Why this

File: app/uk/gov/hmrc/gform/formtemplate/FormTemplateValidator.scala (hunk 1 above)
Comment (DursunGundogan) [reply app/uk/gov/hmrc/gform/formtemplate/FormTemplateValidator.scala:794]: We should check the empty welsh label according to AC1 in [GFORMS-2885](https://jira.tools.tax.service.gov.uk/browse/GFORMS-2885). It does not allow us to post a form if section or page with a title of "cy": "" where the label is not present

Comment (josef-vlach) [reply app/uk/gov/hmrc/gform/formtemplate/FormTemplateValidator.scala:794]: I remember I've posted this: https://github.com/hmrc/gform/pull/1212#pullrequestreview-2303534860
was there some reason why this wasn't implemented and AC1 dropped?

**Review by josef-vlach — APPROVED** [review]

**Pull request merged**